docs_app/src/app/custom-elements/code/code.component.ts

```diff
@@ -147,6 +158,16 @@ export class CodeComponent implements OnChanges {
     return this.codeContainer.nativeElement.textContent;
   }
 
+  /** Extracts html placed in the `// html: ` comment in the code.  */
+  private getHtmlFromCode(code: string): string {
+    const pattern = new RegExp('// html: (.*)');
```

Comment:
Just a note: This assumes there is only one `// html` occurrence in the code and that it does not span multiple lines.